README.md

```diff
@@ -119,7 +119,7 @@ To import pod you should add string:
 - For `Swift` projects:
 
 
-        import CoreActionSheetPicker
+        import ActionSheetPicker_3_0

```

Comment:
there is so weird..I look for the issue about `swift import`,,seem `import CoreActionSheetPicker` still work for other, however `import ActionSheetPicker_3_0` just work for me